lib/pages/main/home_page.dart

```diff
@@ -0,0 +1,385 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:app_tracking_transparency/app_tracking_transparency.dart';
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:mosquito_alert_app/api/api.dart';
+import 'package:mosquito_alert_app/pages/forms_pages/adult_report_page.dart';
+import 'package:mosquito_alert_app/pages/forms_pages/biting_report_page.dart';
+import 'package:mosquito_alert_app/pages/forms_pages/breeding_report_page.dart';
+import 'package:mosquito_alert_app/pages/info_pages/info_page.dart';
+import 'package:mosquito_alert_app/pages/main/components/custom_card_widget.dart';
+import 'package:mosquito_alert_app/pages/my_reports_pages/my_reports_page.dart';
+import 'package:mosquito_alert_app/utils/MyLocalizations.dart';
+import 'package:mosquito_alert_app/utils/UserManager.dart';
+import 'package:mosquito_alert_app/utils/Utils.dart';
+import 'package:mosquito_alert_app/utils/style.dart';
+import 'package:mosquito_alert_app/utils/version_control.dart';
+
+class HomePage extends StatefulWidget {
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  String? userName;
```

Comment:
Is this used?

Reply:
That's not used but it was in the code from before. Maybe it's appropriate that in this PR we focus on the drawer and then on the PR of "homepage with 3 buttons" I clean dead code like this to keep it more organized

Reply:
You're right! Sorry still sleepy today